pcsx2/Sio.cpp

```diff
@@ -132,12 +119,31 @@ void sioInit()
 	sio.packetsize = 0;
 }
 
-void SIO_FORCEINLINE sioInterrupt()
-{
-	PAD_LOG("Sio Interrupt");
-	sio.StatReg|= IRQ;
-	iopIntcIrq(7); //Should this be used instead of the one below?
-	//psxHu32(0x1070)|=0x80;
+bool isR3000ATest = false;
+
+// Check the active game's type, and fire the matching interrupt.
+// Note that the R3000A's call to interrupts only calls the PS2 based (lack of) delays.
+__fi void sioInterrupt() {
```

Comment:
Well why not use ``SIO_FORCEINLINE``? If its uses are limted, best to remove that macro in my opinion. I'm suggesting to use it as I assume there might be certain cases where it would take different values based on other ``ifdef`` checks.